script.js

```diff
@@ -1,4 +1,220 @@
+  }
+
+  // defining draw condition for reversed SPS
+  if (
+    (input == SCISSORS && computerChoice == SCISSORS) ||
+    (input == PAPER && computerChoice == PAPER) ||
+    (input == STONE && computerChoice == STONE)
+  ) {
+    drawRecord = drawRecord += 1;
+    return `X<br>${defaultIntroMessage}<br>X<br>It's a DRAW, ${userName}! <br><br> Your choice: ${input} <br> vs <br> Computer choice: ${computerChoice} <br> You won ${winRecord}/${totalTries} turns and you have ${drawRecord}  draws so far<br><br><br>Pls try again <br><br> ${defaultClosingMessage}`;
+  }
+
+  return myOutputValue;
+};
+
+// Super Game MODE
+var playSuperGame = function (userName, input) {
+  var myOutputValue = ``;
+  var computerChoice = sPS();
+  // validation to inform user can only input SPS
+  if (!(input == SCISSORS || input == PAPER || input == STONE)) {
+    return `Pls enter scissors, paper or stone!`;
+  }
+  if (input == computerChoice) {
+    return `Congrats ${userName}, You Won! You chose ${input}, secret word is ${computerChoice}<br><br>${defaultClosingMessage}`;
+  }
+
+  if (!(input == computerChoice)) {
+    return `Sorry! ${userName}, You Lost! You chose ${input}, secret word is ${computerChoice}<br><br>${defaultClosingMessage}`;
+  }
+  return myOutputValue;
+};
+
+// refactor korean SPS
+var playKoreanSPS = function (userName, input) {
+  var myOutputValue = ``;
+  var computerChoice = sPS();
+  // var winnerOfThisRound = ``;
+  // validation to inform user can only input SPS
+  if (!(input == SCISSORS || input == PAPER || input == STONE))
+    return `Pls enter scissors, paper or stone!`;
+
+  // defining win conditions for Korean SPS
+  if (
+    (input == SCISSORS && computerChoice == PAPER) ||
+    (input == PAPER && computerChoice == STONE) ||
+    (input == STONE && computerChoice == SCISSORS)
+  ) {
+    winnerOfThisRound = userName;
+    return `K<br>${defaultIntroMessage}K <br>Congrats ${userName}, you won this turn!<br> <br> Your choice: ${input} <br> vs <br> Computer choice: ${computerChoice} <br><br><br>Cheers! <br>Most recent winner is ${winnerOfThisRound}<br> ${defaultClosingMessage}`;
+  }
+
+  // defining lose conditions for Korean SPS
+  if (
+    (input == SCISSORS && computerChoice == STONE) ||
+    (input == PAPER && computerChoice == SCISSORS) ||
+    (input == STONE && computerChoice == PAPER)
+  ) {
+    winnerOfThisRound = `computer`;
+    return `K<br>${defaultIntroMessage}K<br> Oops ${userName}, you lost this turn! <br> <br> Your choice: ${input} <br> vs <br> Computer choice: ${computerChoice}<br><br><br> Better luck next time!<br> Most recent winner is ${winnerOfThisRound}<br> ${defaultClosingMessage}
+    `;
```

Comment:
Might want to consider making the win/loss evaluation a function so that it can be reused for playSPS and playKoreanSPS modes